tests/integration/test_dependencies_compatibility_protobuf.py

```diff
@@ -356,3 +356,52 @@ async def test_protobuf_schema_compatibility_dependencies2(registry_async_client
     )
     assert res.status_code == 200
     assert res.json() == {"is_compatible": False}
+
+
+async def test_protobuf_schema_references_values(registry_async_client: Client) -> None:
```

Comment:
suggestion: This test seem to describe a few different expectations, baked into one test case. Can we break this apart into test with descriptive names, such as "returns bad request when {x,y}" and "returns http OK when z"? I find that greatly helps understanding original intent when debugging test failures.

Reply:
I separated rejected and valid values, ie. two test cases.

Reply:
I think this is OK, it's kind of a pet peeve of mine anyway. In my ideal world each case would have it's own descriptive name, so that one could deduce _"ah, we're expecting a rejection here because [some good reason]"_.